focus_areas/code_development.md

```diff
@@ -132,7 +132,7 @@ The name/question pairs listed are not meant to represent a fully comprehensive
 The name/question pairs are intended to be a starting point for CHAOSS-related software. It is expected that this list will evolve based on the ability (or inability) of software to successfully implement the specific name/question pairs.
 
 **Background:**
-The name/question pairs have been identified based CHAOSS-related outreach activities. We thank everyone who participated.
+The name/question pairs have been identified based on CHAOSS-related outreach activities. We thank everyone who participated.
```

Comment:
I have a doubt here. Is this change valid?

Reply:
LGTM

Reply:
I am asking this because I see this in almost every file of the focus_areas directory. So, I was thinking whether it is the error in the template which is inherited in all the metrics or I have mistaken.